Revert to old abbreviations for day of week and months.

diff --git a/localedata/locales/fr_FR b/localedata/locales/fr_FR
index ffa6513..29187ae 100644
--- a/localedata/locales/fr_FR
+++ b/localedata/locales/fr_FR
@@ -94,13 +94,10 @@ grouping                  3
 END LC_NUMERIC
 
 LC_TIME
-abday   "<U0064><U0069><U006D><U002E>";/
-        "<U006C><U0075><U006E><U002E>";/
-        "<U006D><U0061><U0072><U002E>";/
-        "<U006D><U0065><U0072><U002E>";/
-        "<U006A><U0065><U0075><U002E>";/
-        "<U0076><U0065><U006E><U002E>";/
-        "<U0073><U0061><U006D><U002E>"
+abday   "<U0064><U0069><U006D>";"<U006C><U0075><U006E>";/
+        "<U006D><U0061><U0072>";"<U006D><U0065><U0072>";/
+        "<U006A><U0065><U0075>";"<U0076><U0065><U006E>";/
+        "<U0073><U0061><U006D>"
 day     "<U0064><U0069><U006D><U0061><U006E><U0063><U0068><U0065>";/
         "<U006C><U0075><U006E><U0064><U0069>";/
         "<U006D><U0061><U0072><U0064><U0069>";/
@@ -108,18 +105,12 @@ day     "<U0064><U0069><U006D><U0061><U006E><U0063><U0068><U0065>";/
         "<U006A><U0065><U0075><U0064><U0069>";/
         "<U0076><U0065><U006E><U0064><U0072><U0065><U0064><U0069>";/
         "<U0073><U0061><U006D><U0065><U0064><U0069>"
-abmon   "<U006A><U0061><U006E><U0076><U002E>";/
-        "<U0066><U00E9><U0076><U0072><U002E>";/
-        "<U006D><U0061><U0072><U0073>";/
-        "<U0061><U0076><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0069>";/
-        "<U006A><U0075><U0069><U006E>";/
-        "<U006A><U0075><U0069><U006C><U002E>";/
-        "<U0061><U006F><U00FB><U0074>";/
-        "<U0073><U0065><U0070><U0074><U002E>";/
-        "<U006F><U0063><U0074><U002E>";/
-        "<U006E><U006F><U0076><U002E>";/
-        "<U0064><U00E9><U0063><U002E>"
+abmon   "<U006A><U0061><U006E>";"<U0066><U00E9><U0076>";/
+        "<U006D><U0061><U0072>";"<U0061><U0076><U0072>";/
+        "<U006D><U0061><U0069>";"<U006A><U0075><U006E>";/
+        "<U006A><U0075><U0069>";"<U0061><U006F><U00FB>";/
+        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+        "<U006E><U006F><U0076>";"<U0064><U00E9><U0063>"
 mon     "<U006A><U0061><U006E><U0076><U0069><U0065><U0072>";/
         "<U0066><U00E9><U0076><U0072><U0069><U0065><U0072>";/
         "<U006D><U0061><U0072><U0073>";/
